files/files.go

```diff
@@ -359,3 +384,79 @@ func TempFileName(prefix, suffix string) string {
 	rand.Read(randBytes)
 	return filepath.Join(os.TempDir(), prefix+hex.EncodeToString(randBytes)+suffix)
 }
+
+//	Resolve File
+//
+//	If file location passed as argument is a valid,
+//	then it returns the contents of the file.
+//	Otherwise, an error.
+func ResolveFile(file string) (string, error) {
+
+	var payload string
+
+	//	Ensure it is a valid file path,
+	//	by ensuring there an extension suffixed at the end.
+	//
+	//	This will validate both a file in the filesystem,
+	//	as well as the URL source for a file.
+	if filepath.Ext(file) == "" {
```

Comment:
Why does a file or URL need an extension? 

`$value == 1.00` is not a valid path or URL, yet would pass this test?

And likewise 
`/etc/hosts` is a valid path ?

Reply:
Aah, I only added the extension check to return an error if its a directory instead of a file. I totally forgot about files like `/etc/hosts` with no extensions. Let me improve this.